### PR TITLE
Set down baseline info for Factions

### PR DESCRIPTION
**Pull request by CraftmasterMatt**
This serves as a foundation for information and lore about the various factions, such as NT, Syndicate, Changelings, and the Wizard Federation.

Changes:
Adds a 'Factions' folder for dedicated lore in regards to factions- each faction should be given its own folder.
Lore that pertains to multiple factions simultaneously should be put in a 'misc' folder
Adds a document with general information about the factions- how they interact with NT and whether they are friendly, neutral, or hostile; this page can be updated with additional factions (such as the Ashwalkers, Free Golems, cult of Nar'Sie, etc) as needed.

Reason for adding:
This will assist in the development of Beestation's Lore by allowing for a place for lore to be written in regards to each faction. It will also be easily expandable in the future.
In other words, it helps us make our lore more distinct from NV.

### CODE REVIEW

File: Public Canon/General List of Factions.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+--- GENERAL LIST OF FACTIONS ---
+
+This list encompasses general information about the various factions of Beestation's Lore.
+If a faction is added to the lore, make sure to add them to the list.
+
+--- NANOTRASEN ---
+
+An intergalactic megacorporation with vast reach and deep pockets.
+Their current focus is in regards to a strange material known as 'Plasma' that can only be found on the planet 'Lavaland'.
+To further their research, they have built two space stations in orbit of Planet Lavaland.
```
[the remark this answers is not in the document]
Comment (Vexylius) [reply Public Canon/General List of Factions.txt:10]: Also, about those "two" stations, that needs to be reviewed too. (Why is it called 13? Isn't there supposed to be like 11 more of them? Why are they both called 13).

File: Public Canon/General List of Factions.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Intel on the organization that refers to themselveas as 'The Syndicate' is sparse and often conflicting.
+Some intelligence claims that they are a privatized union, others claim that they are a separate, rival megacorporation.
+
+The only confirmed intel is this; they have sleeper agents that have infiltrated Nanotrasen in nearly every department,
+awaiting their orders. Such commands may be something as simple as stealing documents or objects, while other objectives may include assassintation of personnel.
+They also have dedicated Nuclear Operatives, heavily armed and with a single-minded dedication to obtaining one of Nanotrasen's nuclear authentication disks
+in order to detonate a nuclear bomb aboard the station, and they are willing to ruthlessly and brutally cleave a path through anyone and anything that gets in their way.
+
+One thing is clear; they are the sworn enemies of Nanotrasen, and they are dedicating every resource at their disposal toward its destruction.
+
```
Comment (Vexylius) [inline Public Canon/General List of Factions.txt:26]: I'd completely change the Syndie part to have stuff that is more... confirmed/sure, but that might be done later I guess

File: Public Canon/General List of Factions.txt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Utilizing their ability to copy genetic information and become a spitting image of whomever they have copied, they are often used as infiltrators
+by the Syndicate; but other factions may make use of them as well, and some changelings may oppose Nanotrasen for reasons of their own.
+
+Their unique anatomy and their incredible regenerative abilities makes them a force to be reckoned with; the mere mention
+of Changelings or 'lings' is enough to sow panic and mistrust among the crew.
+
+
+--- THE WIZARD FEDERATION ---
+
+The ancient art of magic, thought long ago to be nothing more than fantasy, has proven itself to be real in a very, very potent way.
```
Comment (Vexylius) [inline Public Canon/General List of Factions.txt:40]: Personally I'd replace that "magic" with "highly advanced technology of some superancient origin" and make up some story about those wizards discovering some ancient tech relic that lets them manipulate the fabric of reality itself around them. It might be tied to cults too

File: Public Canon/To-Do.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+### TO DO ###
+
+!Assemble complete list of factions
+ !Write detailed lore pages about each faction
```
Comment (Vexylius) [inline Public Canon/To-Do.txt:4]: Important to add that it mustn't be in a form of a story, but more of a collection of facts in a chronological order. Only the important stuff

File: Public Canon/General List of Factions.txt [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+This list encompasses general information about the various factions of Beestation's Lore.
+If a faction is added to the lore, make sure to add them to the list.
+
+--- NANOTRASEN ---
+
+An intergalactic megacorporation with vast reach and deep pockets.
+Their current focus is in regards to a strange material known as 'Plasma' that can only be found on the planet 'Lavaland'.
+To further their research, they have built two space stations in orbit of Planet Lavaland.
+These stations are codenamed 'GOLDEN' and 'SAGE'.
```
Comment (Vexylius) [inline Public Canon/General List of Factions.txt:11]: As above, I'm not sure about that, might wanna discuss it on discord

**Review by Vexylius — CHANGES_REQUESTED** [review]

**Comment by Vexylius** [comment]
ah, I requested changes but meant to comment, oh well.

**Comment by Vexylius** [comment]
Also, there should be the same General List of Factions file in /Secret Canon with EVERYTHING that's agreed to be a fact in BeeLore. That's what that folder is there for /Public Canon is for more stripped down versions and confirmed stories that are fully reviewed.

**Comment by CraftmasterMatt** [comment]
Understood, I'll see what sort of changes I can make.

File: Public Canon/General List of Factions.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (CraftmasterMatt) [reply Public Canon/General List of Factions.txt:10]: There's a possibility that the two stations could both be called 13, as they might be the 'latest' in Nanotrasen's lineup of science stations. 

According to a conversation on Discord, NT has 2 space stations in orbit of Lavaland, which are Golden and Sage. The two stations allow for, in theory, twice the amount of resources to be devoted to research of Plasma and Bluespace Technology.  In practice, they are separate stations so that if _something_ unfortunate were to happen to one, they could shift operations to the other.

File: Public Canon/General List of Factions.txt (hunk 2 above)
Comment (CraftmasterMatt) [reply Public Canon/General List of Factions.txt:26]: Yes, this is basically what is 'known' about the Syndicate. It is left intentionally vague with the few known factors present, so that we can take the Syndicate in any direction we'd like. Perhaps NT is the enemy, and the Syndicate is good? It's all up for debate and discussion.

File: Public Canon/General List of Factions.txt (hunk 3 above)
Comment (CraftmasterMatt) [reply Public Canon/General List of Factions.txt:40]: Hmm, perhaps they've tapped into the power of some neutral entity along the lines of Rat'var or Nar'Sie, but aren't so desperate to bring the entity into the present world that they're about to breach the veil in order to do so. Of course, I like the idea of something completely alien and un-scientific such as magic being present that may or may not end up being the subject of further research from Nanotrasen.

File: Public Canon/To-Do.txt (hunk 4 above)
Comment (CraftmasterMatt) [reply Public Canon/To-Do.txt:4]: Noted, will be adjusted shortly

File: Public Canon/General List of Factions.txt (hunk 3 above)
Comment (CraftmasterMatt) [reply Public Canon/General List of Factions.txt]: It has been adjusted, see newest commit

File: Public Canon/General List of Factions.txt (hunk 5 above)
Comment (CraftmasterMatt) [reply Public Canon/General List of Factions.txt]: It has been adjusted, see newest commit

File: Secret Canon/readme.txt [hunk 6]
```diff
@@ -1,3 +1,13 @@
...
+to further lore additions and changes. 
+NOT ALL OF THIS INFORMATION IS PUBLIC- Public knowledge is allowed to be incorrect due to lack of information.
+This is everything that has been determined to be canonical, and may include information that is not public knowledge.
+
+Referring to this should only be used if there is a conflict between something that is attempting to be made canon
+and actual canon knowledge (which can be found here). In case of conflict, generally established canon will win over
+attempted alterations, barring extreme circumstances.
+
+Referring to things in this area that is -NOT- public knowledge (barring permissable exceptions) 
+is considered metaknowledge and is frowned upon significantly.
```
Comment (Vexylius) [inline Secret Canon/readme.txt:13]: Love the entirety of it

File: Public Canon/readme.txt [hunk 7]
```diff
@@ -1,3 +1,6 @@
-####################
-# Work in progress #
-####################
\ No newline at end of file
+### PUBLIC CANON ###
+
+These documents are common knowledge among those in the BEESTATION universe.
+In other words, this is knowledge that has been made available and is known by most people in general.
+Due to the nature of these documents, they may or may not be completely accurate. This is just a collection of
+knowledge that has been obtained. It may be incorrect in-universe due to a lack of complete information.
```
Comment (Vexylius) [inline Public Canon/readme.txt:6]: I'd also add that this directory is for documents, stories, fiction from the this universe and rough timelines, to more show what should be in this folder generally.
Also I'd suggest that if people are working on some big story or some other project like that, they are advised to put it in /Not classified first, and then it may be moved here, at least that is how I'd prefer it to work.
Also by the way /Not classified is for everything relevant, even if non-canon, so NSV lore and even irl facts I guess.

File: Public Canon/General List of Factions.txt [hunk 8]
```diff
@@ -5,11 +5,10 @@ If a faction is added to the lore, make sure to add them to the list.
 
 --- NANOTRASEN ---
 
-An intergalactic megacorporation with vast reach and deep pockets.
-Their current focus is in regards to a strange material known as 'Plasma' that can only be found on the planet 'Lavaland'.
-To further their research, they have built two space stations in orbit of Planet Lavaland.
-These stations are codenamed 'GOLDEN' and 'SAGE'.
-Due to their influence, there are many that would wish them harm.
+A group of some sort with interest in researching advanced technologies.
```
Comment (Vexylius) [inline Public Canon/General List of Factions.txt:8]: I'd stay with that it's a megacorporation, but I'd expand on that the fact that it's so big makes it function kinda like a government, because they have their own law, their territory and citizens.

File: Not classified/readme.txt [hunk 9]
```diff
@@ -1,3 +1,6 @@
-####################
-# Work in progress #
-####################
\ No newline at end of file
+### NOT CLASSIFIED ###
+
+These documents are PUBLIC knowledge, but not COMMON knowledge among those in the BEESTATION universe.
+In other words, this is knowledge that has been made available, but is not known by everyone.
+Due to the nature of these documents, they may or may not be completely accurate. This is just a collection of
+knowledge that has been obtained. It may be incorrect and is subject to change.
```
Comment (Vexylius) [inline Not classified/readme.txt:6]: As I've said in the comment to /Public Canons readme it's more of a dump for everything that is relevant to the project but non canon and also WIP projects that are not yet canon. Basically if you don't know where to put something you put it here. Stuff always can be moved if needed

File: TODO.txt [hunk 10]
```diff
@@ -17,4 +17,12 @@
  >Put NSV lore in the NSV lore folder
  
 !Make some writing guidelines for this project
- !Put some guidelines them in those readme.txt files and write what's supposed to be in those folders
\ No newline at end of file
+ !Put some guidelines them in those readme.txt files and write what's supposed to be in those folders
+	// Work in Progress. -CraftmasterMatt
+
```
Comment (Vexylius) [inline TODO.txt:22]: might actually add a low priority one to convert those readme.txts into readme.md so they can be more easily viewable

**Review by Vexylius — CHANGES_REQUESTED** [review]
Add some adjustments and tell me when you're ready to merge it
